### PR TITLE
Story/dfc 10213 job profile current opportunities

### PR DESCRIPTION
**Pull request by Gurnaik-Randhawa**
Initial part one of the story DFC-10213 to build solution and implement tasks and test as outlined in the acceptance criteria.

### CODE REVIEW

File: DFC.App.JobProfile.CurrentOpportunities.Data/Models/CurrentOpportunitiesSegmentData.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using DFC.App.JobProfile.CurrentOpportunities.Data.Contracts;
+using System.Collections.Generic;
+
+namespace DFC.App.JobProfile.CurrentOpportunities.Data.Models
+{
+    public class CurrentOpportunitiesSegmentData : IDataModel
```
Comment (ilyasdhin) [inline DFC.App.JobProfile.CurrentOpportunities.Data/Models/CurrentOpportunitiesSegmentData.cs]: Can you rename to CurrentOpportunitiesSegmentDataModel

File: DFC.App.JobProfile.CurrentOpportunities.UnitTests/DFC.App.JobProfile.CurrentOpportunities.UnitTests.csproj [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    <None Remove="TestHelpers\**" />
+    <None Remove="ViewTests\**" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <AdditionalFiles Include="..\stylecop.json" Link="stylecop.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="FakeItEasy" Version="5.2.0" />
```
Comment (ilyasdhin) [inline DFC.App.JobProfile.CurrentOpportunities.UnitTests/DFC.App.JobProfile.CurrentOpportunities.UnitTests.csproj]: You seem to have multiple versions of the same package - check and consolidate the nuget packages

File: DFC.App.JobProfile.CurrentOpportunities/DFC.App.JobProfile.CurrentOpportunities.csproj [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  </PropertyGroup>
+
+  <ItemGroup>
+    <AdditionalFiles Include="..\stylecop.json" Link="stylecop.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="7.0.0" />
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.7.1" />
+    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.2.6" />
```
Comment (ilyasdhin) [inline DFC.App.JobProfile.CurrentOpportunities/DFC.App.JobProfile.CurrentOpportunities.csproj]: Dont use a specific version of this package.

File: DFC.App.JobProfile.CurrentOpportunities.Data/Models/CurrentOpportunitiesSegmentData.cs (hunk 1 above)
Comment (Gurnaik-Randhawa) [reply DFC.App.JobProfile.CurrentOpportunities.Data/Models/CurrentOpportunitiesSegmentData.cs]: Done

File: DFC.App.JobProfile.CurrentOpportunities.UnitTests/DFC.App.JobProfile.CurrentOpportunities.UnitTests.csproj (hunk 2 above)
Comment (Gurnaik-Randhawa) [reply DFC.App.JobProfile.CurrentOpportunities.UnitTests/DFC.App.JobProfile.CurrentOpportunities.UnitTests.csproj]: Consolidated - think the version went up while I was putting the various parts of the solution together!

**Pull request merged**